### ISSUE TITLE
Readme documents "subscribed" event that has no implementation

### ISSUE THREAD

**Issue by maybejustmaybe** [ipfs-shipyard/ipfs-pubsub-room#76]
The pubsub room never emits a "subscribed" event. I don't see any implementation for sending this event.

Is there a plan to start emitting this event? If not, should we remove it from the readme?

**Comment by DougAnderson444** [comment]
Yeah, I wondered why it wasn't working, lol. Seeing the same thing
